### PR TITLE
Fixed links to ./public/css/styles.css and ./public/img/logo.png

### PR DESCRIPTION
**Pull request by TeaBizzy**
In reference to issue #139. Fixes the links in `index.html` to the `styles.css` & `logo.png`. The webpage now displays correctly.

Please let me know of any feedback or issues with this PR.

Cheers!

### CODE REVIEW

**Review by TAKANOME-DEV — COMMENTED** [review]
For some reason, it doesn't show up on Gitpod, it's as if the link to the CSS doesn't work. @TeaBizzy did you run the server and check if it still works?

**Comment by TeaBizzy** [comment]
@TAKANOME-DEV it works with the server up for me when I open index.html via the file. Not sure how to navigate to this through the server?

**Comment by TAKANOME-DEV** [comment]
> @TAKANOME-DEV it works with the server up for me when I open index.html via the file. Not sure how to navigate to this through the server?

Visit `localhost:8000` in your browser and let me know if everything works

File: src/controller/mainController.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const path = require('path');
 
 const controller = {
   home: (req, res) => {
-    return res.sendFile(path.resolve('./src/views/index.html'));
+    return res.sendFile(path.resolve('./public/views/index.html'));
```
Comment (TAKANOME-DEV) [inline src/controller/mainController.js:5]: The public folder does not have `views`, it's only located in the `src`

File: public/views/index.html [hunk 2]
```diff
@@ -32,12 +32,12 @@
       href="https://fonts.googleapis.com/css2?family=Poppins&display=swap"
       rel="stylesheet"
     />
-    <link rel="stylesheet" href="../../public/css/styles.css" />
+    <link rel="stylesheet" href="../css/styles.css" />
```
Comment (TAKANOME-DEV) [inline public/views/index.html:35]: It would be awesome if you remove the double dots `..`

File: public/views/index.html [hunk 3]
```diff
@@ -32,12 +32,12 @@
...
   </head>
   <body>
     <nav>
       <div class="logo">
-        <a href="/"><img src="../../public/img/logo.png" alt="logo" /></a>
+        <a href="/"><img src="../img/logo.png" alt="logo" /></a>
```
Comment (TAKANOME-DEV) [inline public/views/index.html:40]: And here also `..`

**Review by TAKANOME-DEV — COMMENTED** [review]
Sorry about all the comments :pray:  just wanted to double check if everything is ok

File: public/views/index.html (hunk 2 above)
Comment (TAKANOME-DEV) [reply public/views/index.html:35]:
```html 
 <link rel="stylesheet" href="/css/styles.css" />
````

File: public/views/index.html (hunk 3 above)
Comment (TAKANOME-DEV) [reply public/views/index.html:40]:
```html
 <a href="/"><img src="/img/logo.png" alt="logo" /></a>
```

**Comment by TeaBizzy** [comment]
@TAKANOME-DEV I have the issue sorted out, but I made some errors with my git structure. So I'm thinking about closing this PR and reopening it. 
Problem is, for me to fix these links it requires #141 to be merged with master as I can't start the server without those errors being fixed. Ideally I'd prefer to open 2. So I'm trying to figure out how to handle both as cleanly as possible.

File: src/controller/mainController.js (hunk 1 above)
Comment (TeaBizzy) [reply src/controller/mainController.js:5]: @TAKANOME-DEV In order for the links to be fixed I have to move /views to /public. I'm working on it now, but see my latest comment. Let me know what you think.

**Comment by TAKANOME-DEV** [comment]
Oh, I get it. I think the easiest way is to wait until #141 gets merged. and all you have to do after is: "git fetch upstream" "git merge upstream/master" and "git push -u origin <this_current_branch_name>"
If you have any issues updating your branch after the Branch #141 merger, please let me know and I will assist you as much as possible.

**Pull request closed without merging**